app/src/main/java/chat/rocket/android/service/observer/MethodCallObserver.java

```diff
@@ -117,10 +118,10 @@ public void onUpdateResults(List<MethodCall> results) {
           } else if (exception instanceof DDPClientCallback.RPC.Timeout) {
             // temp "fix"- we need to rewrite the connection layer a bit
             errMessage = "{\"message\": \"Connection Timeout\"}";
-          } else if (exception instanceof RxWebSocketCallback.Failure) {
```

Comment:
If it is not being used, just remove it... If it is something temporary, add a comment indicating that.